app-emulation/virtualbox/files/virtualbox-7.0.0-fix-compilation.patch

```diff
@@ -0,0 +1,72 @@
+vboximportchecker is not available in Linux
```

Comment:
Upstream? Also, why aren't these in the patchset? (Ignore me if it's because it's easier for juippis to review like this.)

Reply:
I found it, it was being removed by the ebuild.  The rest of the patch consists of trivial fixes.
I'll look into sending patches upstream.